test/tests/api/v2_0/workflows_tests.py

```diff
@@ -0,0 +1,213 @@
+
+@test(groups=['workflows_api2.tests'])
+class WorkflowsTests(object):
+
+    def __init__(self):
+        self.__client = config.api_client
+        self.__task_worker = None
+        self.workflowDict = {
+            "friendlyName": "Shell Commands Hwtest_1",
+            "injectableName": "Graph.post.test",
+            "tasks": [{"taskName": "Task.Trigger.Send.Finish"}]
+        }
+        self.workflowDict2 = {
+            "friendlyName": "Shell Commands Hwtest_2",
+            "injectableName": "Graph.post.test",
+            "tasks": [{"taskName": "Task.Trigger.Send.Finish"}]
+        }
+
+    @test(groups=['delete_all_active_workflows_api2'])
+    def delete_all_active_workflows(self):
+        """Testing node DELETE:/nodes/identifier/workflows/active"""
+        Api().nodes_get_all()
+        nodes = loads(self.__client.last_response.data)
+        for node in nodes:
+            if node.get('type') == 'compute':
+                id = node.get('id')
+                assert_not_equal(id,None)
+                try:
+                    Api().nodes_del_active_workflow_by_id(id)
+                except rest.ApiException as err:
+                    LOG.warning(err)
+
+    @test(groups=['workflows_get_api2'], depends_on_groups=['delete_all_active_workflows_api2'])
+    def test_workflows_get(self):
+        """ Testing GET:/workflows"""
+        Api().workflows_get()
+        assert_equal(200,self.__client.last_response.status)
+        assert_not_equal(0, len(json.loads(self.__client.last_response.data)),
+                         message='Active workflows list was empty!')
+
+    @test(groups=['workflows_post_api2'], depends_on_groups=['delete_all_active_workflows_api2'])
+    def test_workflows_post(self):
+        """Testing POST:/workflows"""
+        Api().workflows_post(body={"name": 'Graph.noop-example'})
+        assert_equal(201, self.__client.last_response.status)
+        rawj = json.loads(self.__client.last_response.data)
+        instance_id = rawj.get('instanceId')
+        assert_is_not_none(instance_id)
+        assert_equal('Graph.noop-example', str(rawj['definition'].get('injectableName')))
+
+    @test(groups=['workflows_get_id_api2'], depends_on_groups=['workflows_get_api2'])
+    def test_workflows_id_get(self):
+        """ Testing GET:/workflows/identifier"""
+
+        # Getting the identifier of the first workflow in order to validate the get-id function
+        Api().workflows_get()
+        rawj = json.loads(self.__client.last_response.data)
+        instance_id = rawj[0].get('id')
+        assert_is_not_none(instance_id)
+        Api().workflows_get_by_id(instance_id)
+        assert_equal(200,self.__client.last_response.status)
+
+    @test(groups=['workflows_get_id_api2'],depends_on_groups=['workflows_get_api2'])
+    def test_negative_workflows_id_get(self):
+        """ Negative Testing GET:/workflows/identifier"""
+        try:
+            Api().workflows_get_by_id("WrongIdentifier")
```

Comment:
We should add an assertion in the try block after the Api call.  We want to make sure we get an error, so the test should fail if an exception is not raised.